yarn.lock

```diff
@@ -16,11 +16,23 @@
   dependencies:
     "@babel/highlight" "^7.12.13"
 
+"@babel/code-frame@^7.14.5":
```

Comment:
Something looks wrong in the `yarn.lock` as all the babel dependencies are now duplicated.
How do you update the packages?

I would recommend running `yarn upgrade` to get rid of those duplicated packages.

Reply:
I usually run “ yarn upgrade-interactive --latest”, followed by “yarn install” to take care of any dependencies I manually update.